src/app/species/page.tsx

```diff
@@ -0,0 +1,43 @@
+"use client";
+import { Input } from "@/components/ui/input";
+import { findSpecies, SpeciesGetResponse } from "../actions";
+import { useState } from "react";
+import { Badge } from "@/components/ui/badge";
+
+export default function SpeciesPage() {
+  const [speciesSearchResponse, setSpeciesSearchResponse] =
+    useState<SpeciesGetResponse>([]);
+
+  const handleSearch = async (query: string) => {
+    const species = await findSpecies(query);
+    setSpeciesSearchResponse(species);
+  };
+  return (
+    <div>
+      <h1>Species</h1>
+      <form className="mb-4">
+        <Input
+          className="shadow-sm"
+          placeholder="Search for a species"
```

Comment:
Of course it's more of a design question, but we can make some small tweaks here so the language marries up with the home page.

(this suggestion also requires the Label component import)

```suggestion
      <h1 className="mb-12 text-4xl">
        Species{" "}
        <span role="img" aria-label="magnifying-glass">
          🔎
        </span>
      </h1>
      <form className="mb-4  flex flex-col gap-3">
      <Label htmlFor="speciesSearch" className="gap-2">
          Search for a Species:
        </Label>
        <Input
          className="shadow-sm"
          id="speciesSearch"
```
<img width="428" alt="Screenshot 2024-08-08 at 09 31 31" src="https://github.com/user-attachments/assets/8bee8f56-efb4-4199-8e57-f1dabfb79242">
<img width="430" alt="Screenshot 2024-08-08 at 09 31 22" src="https://github.com/user-attachments/assets/82d4b523-fe63-4107-9aa9-a964810a24ab">

Reply:
Yeah good shout! 

I'll make a ticket and we can come back and line this up.

Honestly most of this will likely change a lot as we figure out exactly the sort of user experience we want to prioritise!

Reply:
[Added a ticket for it](https://www.notion.so/Species-Search-Page-UI-alignment-bf1d746bc2f044de8bcdb93fdde2eba5?pvs=4)